tools/merge_schedule

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/perl
+
+# Tool to merge YAML_SCHEDULE_DEFAULT  with YAML_SCHEDULE_FLOWS
+# and YAML_SCH§EDULE.
+
+# So far all pointers should be provided as environment
```

Comment:
this comment shouldn't be hear, should be on top the declaration of the env variables.